test/e2e/e2e.go

```diff
@@ -656,7 +588,7 @@ func restartOVNKubeNodePodsInParallel(clientset kubernetes.Interface, namespace
 	for _, n := range nodeNames {
 		nodeName := n
 		restartFuncs = append(restartFuncs, func() error {
-			return restartOVNKubeNodePod(clientset, ovnNamespace, nodeName)
+			return restartOVNKubeNodePod(clientset, namespace, nodeName)
```

Comment:
namespace fixes could be a separate commit

Reply:
done